lmnet/lmnet/networks/keypoint_detection/lm_single_pose_v1.py

```diff
@@ -0,0 +1,218 @@
+        x = self._space_to_depth(name='space2depth1', inputs=x)
+        x = _lmnet_block('conv2', x, 32, 3)
+        x = _lmnet_block('conv3', x, 32, 3)
+        c1 = _lmnet_block('c1', x, 32, 1)
+
+        x = self._space_to_depth(name='space2depth2', inputs=x)
+        x = _lmnet_block('conv4', x, 32, 3)
+        x = _lmnet_block('conv5', x, 32, 3)
+        x = _lmnet_block('conv6', x, 32, 3)
+        c2 = _lmnet_block('c2', x, 32, 1)
+
+        x = self._space_to_depth(name='space2depth3', inputs=x)
+        x = _lmnet_block('conv7', x, 64, 3)
+        x = _lmnet_block('conv8', x, 64, 3)
+        x = _lmnet_block('conv9', x, 64, 3)
+        c3 = _lmnet_block('c3', x, 32, 1)
+
+        x = self._space_to_depth(name='space2depth4', inputs=x)
+        x = _lmnet_block('conv10', x, 128, 3)
+        x = _lmnet_block('conv11', x, 128, 3)
+        x = _lmnet_block('conv12', x, 128, 3)
+        c4 = _lmnet_block('c4', x, 32, 1)
+
+        x = self._space_to_depth(name='space2depth5', inputs=x)
+        x = _lmnet_block('conv13', x, 256, 3)
+        x = _lmnet_block('conv14', x, 256, 3)
+        x = _lmnet_block('conv15', x, 256, 3)
+        x = _lmnet_block('conv16', x, 256, 3)
+        x = _lmnet_block('conv17', x, 256, 3)
+        c5 = _lmnet_block('c5', x, 32, 1)
+
+        x = connect_block(lateral=c4, up=c5, name="connect_block1")
+
+        if self.stride < 16:
+            x = connect_block(lateral=c3, up=x, name="connect_block2")
+            if self.stride < 8:
+                x = connect_block(lateral=c2, up=x, name="connect_block3")
+                if self.stride < 4:
+                    x = connect_block(lateral=c1, up=x, name="connect_block4")
+
+        x = _lmnet_block('conv_final', x, self.num_joints, 3, activation=None)
+
+        return x
+
+
+class LmSinglePoseV1Quantize(LmSinglePoseV1):
+    """LM original quantized single-person pose estimation network.
+
+    Following `args` are used for inference: ``activation_quantizer``, ``activation_quantizer_kwargs``,
+    ``weight_quantizer``, ``weight_quantizer_kwargs``.
+
+    Args:
+        activation_quantizer (callable): Weight quantizater. See more at `lmnet.quantizations`.
+        activation_quantizer_kwargs (dict): Kwargs for `activation_quantizer`.
+        weight_quantizer (callable): Activation quantizater. See more at `lmnet.quantizations`.
+        weight_quantizer_kwargs (dict): Kwargs for `weight_quantizer`.
+
+    """
+
+    def __init__(
+            self,
+            activation_quantizer=None,
+            activation_quantizer_kwargs=None,
+            weight_quantizer=None,
+            weight_quantizer_kwargs=None,
+            *args,
+            **kwargs
+    ):
+        super().__init__(
+            *args,
+            **kwargs
+        )
+
+        assert weight_quantizer
+        assert activation_quantizer
+
+        activation_quantizer_kwargs = activation_quantizer_kwargs if activation_quantizer_kwargs is not None else {}
```

Comment:
i think there's no need to say `is not None`.
`a = a if a else {}`

Reply:
In all existing config file, `NETWORK.WEIGHT_QUANTIZER_KWARGS = {}`.
```
>>> a = {}
>>> if a:
...     print(a)
...
>>> if a is not None:
...     print(a)
...
{}
```
So `a = a if a else {}` may not work here except we refactor all config files.

Reply:
But both of them are okay here..